skiko/src/commonMain/kotlin/org/jetbrains/skia/Font.kt

```diff
@@ -419,11 +445,22 @@ class Font : Managed {
     fun getBounds(glyphs: ShortArray?, p: Paint?): Array<Rect> {
         return try {
             Stats.onNativeCall()
-            _nGetBounds(
-                _ptr,
-                glyphs,
-                getPtr(p)
-            )
+
+            if (glyphs == null) {
+                emptyArray()
+            } else {
+                withResult(FloatArray(glyphs.size * 4))  {
```

Comment:
Maybe this can be moved to `Rect` like I did for `AnimationFrameInfo`?

Reply:
yep, totally make sense, pushed